config/pipelines.yml

```diff
@@ -129,9 +129,7 @@ default: &default
           value: insert_size
           populate_on_row_type: :well
         Template Prep Kit Box Barcode:
```

Comment:
So based on the changes above this would become
```
Template Prep Kit Box Barcode:
 type: :model
 value: template_prep_kit_box_barcode
 populate_on_row_type: :well
```